### ISSUE TITLE
anisotropy not used correctly in optical_forward_model_mcx_adapter

### ISSUE THREAD

**Issue by mschllnbrg** [IMSY-DKFZ/simpa#139]
In the optical_forward_model_mcx_adapter the _assumed_anisotropy is used instead of the anisotropy. 

self.generate_mcx_bin_input(absorption_cm=absorption_cm,
                                    scattering_cm=scattering_cm,
                                    !!! anisotropy=_assumed_anisotropy, !!!
                                    assumed_anisotropy=_assumed_anisotropy)

Therefore, a wrong reduced scattering is used as the input of MCX.

**Comment by jgroehl** [comment]
Oh my. Good that we spotted this. Can we hotfix this into the master branch, too?

**Comment by mschllnbrg** [comment]
Sounds good from my side!!! And yes, good that we found this bug. :)

**Comment by jnoelke** [comment]
Maybe we should also save the recalculated scattering that is finally used within MCX in the hdf5.

**Comment by jgroehl** [comment]
Good idea. It might also be worth spending time to help Qianqian implement a "proper" (mua, mus, g, n) modelling in mcx and to add refractive index modelling to SIMPA.

**Comment by kdreher** [comment]
Won't be used in future
